spec/martian_robot_spec.rb

```diff
@@ -1,82 +1,105 @@
 $LOAD_PATH << File.expand_path('. ./lib', __FILE__)
```

Comment:
You shouldn't have to loads the path when you using RSpec. Did you had some issues with it?